includes/common.js

```diff
@@ -1,26 +1,26 @@
 // List of standard fields shared across the rest of the project
 
 // From here: https://segment.com/docs/connections/spec/page/
-let PAGE_FIELDS = [
-    "url",
-    "referrer",
-    "title",
-    "name",
-    "search",
-    "path",
-    "context_campaign_content",
-    "context_campaign_medium",
-    "context_campaign_source",
-    "context_campaign_name",
-    "context_campaign_term",
-    "context_campaign_keyword"];
+let PAGE_FIELDS = {
+    url: "url",
+    referrer: "referrer",
+    title: "title",
+    name: "name",
+    search: "search",
+    path: "path",
+    context_campaign_content: "utm_content",
```

Comment:
IMO, this isn't a great idea. For people familiar with the segment tables already, this just makes it a bit harder to use the table we produce. I get they are shorter, but is it worth it?